src/Home.vue

```diff
@@ -11,6 +11,24 @@
       </div>
     </div>
 
+    <div id="videolist">
+      <div class="container">
+        <div class="row justify-content-center">
+          <div class="col col-md-8" >
+            <iframe 
+            width="100%" 
+            height="412em" 
+            src="https://www.youtube-nocookie.com/embed?list=PLCbZc-GgCGzLhkGS5l850tjCQrPUeapK6" 
```

Comment:
Maybe make the link configurable as `yt-link` data which can be accessed as `{{yt-link}}`